### ISSUE TITLE
builder.Prompts.text not working after being hosted remotely

### ISSUE THREAD

**Issue by mcclaskiem** [Microsoft/BotBuilder#2372]
I have a simple bot that fetches news articles based on a user prompt. The entire flow works fine locally using emulator but after being deployed to a server the bot fails when it hits a `builder.Prompts.text` block. Below is my code and you will see a "Asking article count" prompt which is where it stops in flow. 

- Bot shows accepted when testing on the BOT Framework page
- Bot is receiving messages via WebChat and Slack
- Bot also shows 0 issues for each channel after interacting

```
var bot = new builder.UniversalBot(connector);
var intents = new builder.IntentDialog();
bot.dialog('/', intents);

var HHCC = require('./hhcc.js');



intents.matches(/^news/i, [
    function(session) {
        console.log("Intent Given!");
        session.beginDialog('/news');
    },
    function(session, results) {
        session.send('Enjoy reading!');
    }
]);

bot.dialog('/news', [
    function(session) {
        console.log("Asking article count");
        builder.Prompts.text(session, 'How many articles would you like to see?');
    },
    function(session, results) {
        session.sendTyping();
        session.conversationData.count = results.response;
        HHCC.getNews(session.conversationData.count, session, function(newsArticles) {
            newsArticles.forEach(function(newsCard) {
                session.send(newsCard);
            });
            session.conversationData.news = newsArticles;
            console.log(newsArticles);
            session.endDialog();
        });
    }
]);


server.post('/api/messages', connector.listen());
```

Ive checked all logs and can't seem to find any clues as its failing pretty silently.

**Comment by Stevenic** [comment]
@mcclaskiem a couple of quick questions:

- When hosted remotely have you tried connecting to your bot using the emulator? Does that work?
- Have you tried just doing `session.send("I heard %s", results.response);` just to verify you're making it to you next waterfall step?
- You do realize `Prompts.text()` is going to return a `string`? I would have expected you to use `Prompts.number()` which will return a `number`.

**Comment by mcclaskiem** [comment]
@Stevenic I was able to figure out the issue. The .env file was not being read locally or on server by simply dropping in the example provided. I needed to add node_module to allow the bot to read the .env file and ultimately allow for messages to flow back and forth.

You might want to consider changing how the test functionality works in the dashboard since it continued to give me "Accepted" even though it wasn't actually testing against the creds.
